src/Analysis/GWASResultsV2/Views/Execution/Execution.stories.jsx

```diff
@@ -122,3 +122,11 @@ MockedError.parameters = {
     ],
   },
 };
+export const MockedError403Response = MockTemplateFailure.bind({});
+MockedError403Response.parameters = {
+  msw: {
+    handlers: [
+      rest.post('', (req, res, ctx) => res(ctx.delay(800), ctx.status(403))),
```

Comment:
This one is not working at the moment.

Reply:
This has been updated per our discussion.